core/modules/wildcard_match.cc

```diff
@@ -335,7 +335,10 @@ int WildcardMatch::DelEntry(struct WmTuple *tuple, wm_hkey_t *key) {
     tuple->ht.Close();
 
     num_tuples_--;
-    memmove(&tuples_[idx], &tuples_[idx + 1],
+
+    // shift remaing tuples to the left
+    memmove(reinterpret_cast<void*>(&tuples_[idx]),
```

Comment:
Good idea. We would need a way for module writers to somehow set up the appropriate module state, though. I'll write up a todo.